genesis-template.json

```diff
@@ -11,13 +11,31 @@
     "petersburgBlock": 0,
     "istanbulBlock": 0,
     "muirGlacierBlock": 0,
+    "berlinBlock": 13996000,
+    "londonBlock": 22640000,
     "bor": {
-      "period": 1,
-      "producerDelay": 4,
+      "jaipurBlock": 22770000,
```

Comment:
All blocks should be 0. Since new genesis for new chain only and for that all forks will be activated from block 0.